dashboard/package.json

```diff
@@ -62,10 +65,10 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.5",
     "redux-thunk": "^2.2.0",
+    "seamless-immutable": "^7.1.4",
```

Comment:
I also explicitly install 4 or 5 packages which were generating peer dependency warnings when building (afaict, this means you need to explicitly install a version yourself - which I did, but may not be required).